shopinvader_payment/services/abstract_payable_sale.py

```diff
@@ -10,7 +10,7 @@ class AbstractPayableSaleService(AbstractComponent):
     _inherit = "base.shopinvader.service"
     _name = "shopinvader.abstract.payable.sale.service"
 
-    def _get_available_payment_mode(self, sale):
+    def _get_available_payment_method(self, sale):
```

Comment:
you should use plurals... `s/_get_available_payment_method/_get_available_payment_methods/`